package-lock.json

```diff
@@ -13,6 +13,7 @@
       },
       "devDependencies": {
         "@babel/core": "^7.15.0",
+        "@mermaid-js/mermaid-mindmap": "^9.3.0",
```

Comment:
Can you please exclude the package-lock change from the PR? I'll run this myself after merging this

Otherwise the change looks good